### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by sayantanigoswami1996**
New Script added for review :
1.52Wk High India
2.52Wk High US
3.Mutual Fund
4.Stocks
5.US Stocks

### CODE REVIEW

File: Resources/Utils/DesktopWeb/Common.robot [hunk 1]
```diff
@@ -1,155 +1,154 @@
...
+Press Enter Key
+    [Arguments]  ${element}  
+    Press Keys  ${element}  ENTER
+
+Clear Text Field
+    [Arguments]  ${field}
+    Sleep  500ms
+    ${fieldText} =  Get Value  ${field}
+    ${fieldTextLen} =  Get Length  ${fieldText} 
+    Run Keyword If    """${fieldText}""" != ''
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/Common.robot:131]: Is this correct expression?

File: Resources/Utils/DesktopWeb/Common.robot [hunk 2]
```diff
@@ -1,155 +1,154 @@
...
+Resource    ../../../AppLocators/DesktopWeb/InvestLocators/UltraShortLocators.robot
+*** Keywords ***
+
+Launch URL
+    Open Browser  ${URL}  ${BROWSER}  alias=Kuvera
+    # Maximize Browser Window
+    Set Window Size  ${1920}  ${1080}
+    Reload Page
+    Kuvera Web Close Regulatory Disclosure
+    Run Keyword If    "${ENV}" == "prod"  Close Hello Bar
```
Comment (nivethan-moolya) [inline Resources/Utils/DesktopWeb/Common.robot:32]: can we move this hard-coded value into variables?

File: Resources/Utils/DesktopWeb/Common.robot (hunk 1 above)
Comment (PrathijaEtagi) [reply Resources/Utils/DesktopWeb/Common.robot:131]: Yes and workig fine

File: Resources/Utils/DesktopWeb/Common.robot (hunk 2 above)
Comment (PrathijaEtagi) [reply Resources/Utils/DesktopWeb/Common.robot:32]: Fixed

**Pull request closed without merging**